ironfish/src/rpc/routes/chain/followChainStream.ts

```diff
@@ -153,6 +153,7 @@ router.register<typeof FollowChainStreamRequestSchema, FollowChainStreamResponse
             hash: BlockHashSerdeInstance.serialize(transaction.hash()),
             size: getTransactionSize(transaction),
             fee: Number(transaction.fee()),
+            expiration: transaction.expiration(),
```

Comment:
ok nice, thsi also needs to be added to the runtime type validator above for the response.